index.js

```diff
@@ -5,6 +5,10 @@ module.exports = {
 	],
 	ignoreFiles: ['**/*.js', '**/*.ts', '**/*.svg'],
 	rules: {
+		// Stylistic rules conflicting with prettier
+		'scss/operator-no-newline-after': null,
+		'scss/operator-no-newline-before': null,
+
```

Comment:
Isn't `scss/operator-no-unspaced` a formatting rule as well?

Reply:
Ah yes right,  but at least not conflicting